packages/buefy-next/src/components/rate/Rate.vue

```diff
@@ -115,23 +123,23 @@ export default {
             if (this.disabled) return
             this.hoverValue = 0
         },
-        previewRate(index, event) {
+        previewRate(index: number, event: Event) {
```

Comment:
I think you may omit the `event` parameter of this method and append a [`.stop` event modifier](https://vuejs.org/guide/essentials/event-handling#event-modifiers) to the `@mousemove` binding at the following line instead:
https://github.com/ntohq/buefy-next/blob/407a7caed8843f5e6829435e040eb0bde1cd6bec/packages/buefy-next/src/components/rate/Rate.vue#L11

i.e., → `@mousemove.stop="previewRage(item)"`

I usually do not encourage you to change a function signature during the TypeScript migration task. However, regarding this method, there are a couple of good reasons to change it:
- the value of `event` does not matter to the method behavior → removing it won't harm existing applications
- invoking `event.stopPropagation` should not be a concern of the method

Reply:
Sorry, I am wrong. There is a case that does not call `stopPropagation`. I overlooked the following line:
https://github.com/ntohq/buefy-next/blob/407a7caed8843f5e6829435e040eb0bde1cd6bec/packages/buefy-next/src/components/rate/Rate.vue#L119

Reply:
How about this? We do not expect `event` to fully satisfy `Event` anyway.

```ts
preview(index: number, event: { stopPropagation: () => void }) {
```